Dockerfile

```diff
@@ -23,5 +22,7 @@ RUN make realinstall
 
 COPY . /translator
 WORKDIR /translator
-RUN gem install bundler && bundle install --system
```

Comment:
**thought:** Ruby 2.7 already comes with `bundle`. We do not need to install it again.